vello/README.md

```diff
@@ -28,7 +28,8 @@ cargo run -p with_winit
 
 It is used as the rendering backend for [Xilem], a Rust GUI toolkit.
 
-> [!WARNING]
+> ⚠️ WARNING
+>
 > Vello can currently be considered in an alpha state. In particular, we're still working on the following:
```

Comment:
Is this release good enough to call beta? I think probably not, but it's plausible, so I thought I'd check

Reply:
Probably not, pending some more robustness work.